static/app/views/performance/utils/useSpanFieldSupportedTags.tsx

```diff
@@ -5,9 +5,7 @@ import useOrganization from 'sentry/utils/useOrganization';
 import usePageFilters from 'sentry/utils/usePageFilters';
 import {SpanIndexedField} from 'sentry/views/starfish/types';
 
-const omitSupportedTags = [SpanIndexedField.SPAN_AI_PIPELINE_GROUP];
-
-const getSpanFieldSupportedTags = () => {
+const getSpanFieldSupportedTags = omitSupportedTags => {
```

Comment:
Can we rename the variable to something more noun-like? `excludedTags` to match line 35?